Source/relay/TourGuide/Support/IOTMs.ash

```diff
@@ -2,13 +2,32 @@ import "relay/TourGuide/Support/Campground.ash"
 
 boolean [item] __iotms_usable;
 
+void replicaCheck(string iotmName) {
```

Comment:
Code structure feedback, up to you whether it's worth it:
This function would be reusable for checks elsewhere if you made the signature `boolean ownsItemOrReplica(string itemName)` and pushed the __iotms_usable assignment outside of this function. That would make the repeated replicaCheck calls later have more repetition in them, but it'd be easy to factor that out by making a big list of item names and then iterating it similar to what I was doing in the monkey paw tile.

Reply:
i actually had originally written the spec like that prior to doing my first commit on the repo, but found the syntax of the iterating slightly more annoying than i wanted it; this way it felt a little more straightforward. in retrospect i think you're right but i am not positive i want to refactor. i will think on it.

Reply:
not doing this because the refactor would take time and i am too tired. absolutely open to you doing it if you'd like i just want to focus my scant coding energy on the eagle.